packages/core/src/Queue.ts

```diff
@@ -83,12 +88,12 @@ export default class Queue {
     this._capacityMask = (this._capacityMask << 1) | 1
   }
 
-  _shrinkArray () {
+  _shrinkArray (): void {
     this._list.length >>>= 1
     this._capacityMask >>>= 1
   }
 
-  _copyArray () {
+  _copyArray (): Array<A | undefined> {
```

Comment:
This is because underlying `this._list` is also of type `Array<A | undefined>`

Reply:
And `this._list` may contain `undefined`s because of https://github.com/mostjs/core/pull/296/files/017855d69f8390fc990ebc4ba9e521172651089d#diff-e8569040805e38a7296605aeda9326e9R52

Reply:
Ah, right. Thanks for pointing that out.